### ISSUE TITLE
not all wav files supported

### ISSUE THREAD

**Issue by andyweiss** [AWAH-SIP/AWAH-SIP_Codec#68]
wav files with the "fmt" chunk not at offset 12 in the header are not recognized.

It would be also better to be able to support 24bit Files.

**Comment by ajspark4444** [comment]
not at offset 12 ? what do you mean by offset 12 , i know the size is 16 and you get the blockaline by calculating Blockalign = SignificantBitsPerSample / 8 * NumChannels and you can specify the bidepth in the "fmt " format chunk what's the issue ?

**Comment by andyweiss** [comment]
It is about the header of the wav file. Usually the "fmt" format chunk itself is found at the 12th bit in the header. However there are some files where the "fmt" is found at the 648th bit. And these files are not recognized. So its all about the header.

By the way a very warm welcome to our small project ;-)

**Comment by ajspark4444** [comment]
yes i guess sometimes some wave files contain other chunks like the "wavl" ,"slnt" ,"cue " or "plst" so yeah it depand were is the positionning of the "fmt " thank's for the warm welcome
